packages/itwin/grouping-mapping-widget/src/widget/components/GroupAction.tsx

```diff
@@ -349,18 +315,16 @@ const GroupAction = (props: GroupActionProps) => {
         onSave={async () => {
           await save();
         }}
-        onCancel={async () => {
+        onCancel={props.onClickCancel ? async () => {
```

Comment:
So if the `onClickCancel` is not provided, selection is not cleared? Does not look right.

Reply:
In my opinion, cancel can mean abandoning the task completely while "clear" may refer to the action of removing or erasing information or data that was previously entered. I can include this clear action into the onCancel if you feel that it's more intuitive.